gravitee-apim-console-webui/src/management/settings/identity-providers-ng/identity-providers-ng.component.html

```diff
@@ -26,15 +26,29 @@ <h3>Configuration</h3>
   <form *ngIf="identityProvidersForm" [formGroup]="identityProvidersForm" autocomplete="off">
     <mat-card-content>
       <gio-form-slide-toggle formGroupName="forceLogin" class="identity-providers__form__card__form-field">
-        <gio-form-label>Force authentication to access portal</gio-form-label>
+        <gio-form-label
```

Comment:
could you match this display 
```
        <gio-form-slide-toggle
          [matTooltip]="providedConfigurationMessage"
          [matTooltipDisabled]="!isReadonly('logging.audit.enabled')"
          class="api-logging__form__card__form-field"
        >
          <mat-icon *ngIf="isReadonly('logging.audit.enabled')" gioFormPrefix>lock</mat-icon>
          <gio-form-label>Enable audit on API Logging consultation</gio-form-label>
          <mat-slide-toggle
            gioFormSlideToggle
            formControlName="enabled"
            aria-label="Enable audit on API Logging consultation"
            name="enabled"
          ></mat-slide-toggle>
        </gio-form-slide-toggle>
```
to have the same display where this 🔒  is already present

Reply:
I hadn't noticed it before but it would be nice to have the same display everywhere